apps/spruce/src/pages/waterfall/RequesterFilter/index.tsx

```diff
@@ -40,6 +40,7 @@ export const RequesterFilter = () => {
     >
       {commitRequesters.map((requester) => (
         <ComboboxOption
+          key={`${requester}-option`}
```

Comment:
What's this for?

Reply:
Just because there was an error in the console asking to add a `key`!

Reply:
🤦‍♀️